website/docs/configuration/tls.md

```diff
@@ -0,0 +1,48 @@
+---
+title: TLS and certificates
+sidebar_position: 1
+---
+
+## TLS configuration
+
+By default the dashboard will listen on 0.0.0.0:9001 with TLS enabled. A self-signed certificate and key pair are generated when the server starts.
+As the certificate is _self-signed_, Chrome and other browser will show a warning you will have to click through to view the dashboard.
+
+| Parameter           | Type   | Description                                                     | Default |
+| ------------------- | ------ | --------------------------------------------------------------- | ------- |
+| `--no-tls`          | bool   | Disable TLS, access the dashboard on default port via http      | false   |
+| `--tls-private-key` | string | Filename for the TLS certficate, in-memory generated if omitted |         |
+| `--tls-cert-file`   | string | Filename for the TLS key, in-memory generated if omitted        |         |
+| `--host`            | string | Host to listen on                                               | 0.0.0.0 |
```

Comment:
I'm trying to suggest how to change this, and I'm struggling to write something helpful. This is my current best attempt, but it's both too technical and too vague:

By default the dashboard will listen on port 9001 with TLS disabled and without exposing itself to connections from the outside. If you wish to expose it, you first need to set up TLS. You'll need to open the weave-gitops HelmRelease and add some values to the bottom of it. There's two general options.

Either you're already running an ingress controller that terminates TLS for you, in which case you can configure the weave-gitops helm chart to enable an ingress. The exact parameters you need depends on what ingress you use.
```
  values:
    ingress:
      enable: true
      annotations:
        # kubernetes.io/tls-acme: "true"
      hosts:
        # - host: "weave-gitops.mycompany.com",
        #   paths:
        #     - path: "/" 
```

Or you want the dashboard to do TLS termination itself, in which case you need to first create a secret in your cluster:
```
kubectl create secret tls my-tls-secret \
  --cert=path/to/cert/file \
  --key=path/to/key/file
```
and then configure the weave-gitops helm chart to enable TLS termination:
```
  values:
    serverTLS:
      enable: true
      secretName: "my-tls-secret"
```

Either way, when you're done your helm release should look similar to:
```
---
apiVersion: helm.toolkit.fluxcd.io/v2beta1
kind: HelmRelease
metadata:
  name: ww-gitops
  namespace: flux-system
spec:
  chart:
    spec:
      chart: weave-gitops
      sourceRef:
        kind: HelmRepository
        name: ww-gitops
  interval: 1m0s
  values:
    # Either remove serverTLS or ingress:
    serverTLS:
      enabled: true
      # etc
    ingress:
      enabled: true
      # etc
```

Reply:
Thanks. Let me see what I can massage it into...